Makefile

```diff
@@ -1,44 +1,19 @@
 #
 # Makefile for managing docopts build
 #
-# See also: deploy.sh
 
 PREFIX ?= /usr/local
+GOVERSION := $$(go version)
 
 # keep docopts: as first target for development
 
-# govvv define main.Version with the contents of ./VERSION file, if exists
```

Comment:
The `VERSION` is still in use? was used by git tagging (or from or for tagging)

From this doc:
https://github.com/docopt/docopts/blob/master/docs/pre_built_binaries.md#git-tag-a-new-release

It seems it was a trick borrowed from govvv
it outputs
```
govvv -flags
-X main.GitCommit=2b51559 -X main.GitBranch=master -X main.GitState=dirty -X main.GitSummary=v0.6.4-with-no-mangle-double-dash-11-g2b51559 -X main.Version=v0.6.4-with-no-mangle-double-dash -X main.BuildDate=2022-08-24T04:44:03
```

I don't understand yet how the goreleaser handles version name building 

https://goreleaser.com/customization/snapshots/

I don't manipulate go template neither `{{ .Version }}` may I'm lacking knowledge here too.

Reply:
Ohhh, I didn't realize `govvv` used the `VERSION` file like that. I see that in [its docs](https://github.com/ahmetb/govvv#want-to-use-a-different-version) now.

I thought `govvv` used the current Git tag, which is what `goreleaser` does, but guess it's a bit different.

Per [the comment](https://github.com/docopt/docopts/pull/65/files#diff-7326b55c062b0f46fe9e39aace0a25f4515cf206040fb91a6fd2cae839f5e826R20) I left in `.goreleaser.yml`, can see [the docs on `{{ .Version }}`](https://goreleaser.com/cookbooks/using-main.version).
`goreleaser` sets it to the current Git tag (or the snapshot name when using `--snapshot`).

That can be changed if you'd like, but I figure current Git tag and less config by using the defaults makes more sense.

Reply:
> From this doc:
> https://github.com/docopt/docopts/blob/master/docs/pre_built_binaries.md#git-tag-a-new-release

Oh I also totally didn't see this doc, my bad. I read the developer docs but missed that one. I'll have to update that doc now too with all the `goreleaser` and `go.mod` changes

Reply:
Ok, given that the `VERSION` file is used in a lot of places, I'll leave it in. `goreleaser` does not necessarily need changes to adjust since if the tag is prepared first, `goreleaser` will pick it up correctly and match it.

> Very hacky, again, but I should have documented most to the process, as I spent time elsewhere myself and when I come back to some of my previous work, I like to find my own explanation of what I did and why. 😉

Same, I document things quite heavily. But in less hacky fashion 😉 
I missed these files earlier but they do explain your hacky/bespoke process relatively well.

Reply:
I also updated `pre_built_binaries.md` and `release.md` now with all the `goreleaser` changes. Just need to add the file-based changelog / release notes back in now, then should be good to go!